packages/fast-form-generator-react/src/utilities/form-input.style.ts

```diff
@@ -47,7 +66,13 @@ export function applyInputStyle(): ICSSRules<{}> {
         boxShadow: `inset ${toPx(0)} ${toPx(0)} ${toPx(4)} ${colors.boxShadow}`,
```

Comment:
If we had a singular function, we could use it here as well.

Reply:
yep